pkgs/test/cross/default.nix

```diff
@@ -134,6 +134,7 @@ let
     pkgs.pkgsMusl.stdenv
     pkgs.pkgsLLVM.stdenv
     pkgs.pkgsStatic.bash
+    #pkgs.pkgsCross.gnu64_simplekernel.bash   # https://github.com/NixOS/nixpkgs/issues/264989
```

Comment:
Could we use pkgsLLVM or something instead so we don't need to use a new example?

Reply:
> Could we use pkgsLLVM or something instead so we don't need to use a new example?

Unfortunately `pkgsLLVM` doesn't demonstrate the problem:

This will succeed:

@ofborg build pkgsLLVM.bash

This will fail:

@ofborg build pkgsCross.gnu64_simplekernel.bash